README.md

```diff
@@ -1,15 +1,18 @@
 # shibboleth-plugin
 
+### Min. shibboleth-idp version:
+4.3.x
+
 ### Installation:
 1. **Copy the package to the server:**
     - Go to: `/privacyIDEA-dist/target/`.
```

Comment:
This has to be adjusted if the targets folder are not checked in anymore

Reply:
[9ce729e](https://github.com/privacyidea/shibboleth-plugin/pull/7/commits/9ce729e9bf1ece1d87cd5b6a1b7e5a024468e482)